build/import/Packages.targets

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <RestoreSources>
       $(RestoreSources);
+      D:\PROJECT\cps\bin\Debug\packages\nuget;
```

Comment:
ignore this

Reply:
FYI you can do this locally as well via `NuGet.config`. In my `%appdata%\NuGet\NuGet.Config` I have something like:

```xml
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <packageSources>
    <add key="NuGet Local" value="D:\NuGetLocal" />
```